config/services.yaml

```diff
@@ -1,5 +1,6 @@
 parameters:
     env(DEFAULT_LOCALE): 'en'
+    env(INFO_BAR): null
```

Comment:
Not sure this (and the other) should be environment variables. More complicated to process, hence https://github.com/libero/browser/blob/a7b804f25e7dbfa58a2970720ba6ca55e00bebc3/vendor-extra/LiberoPageBundle/src/EventListener/InfoBarListener.php#L31-L33

Neither are things that should be changeable at runtime.

/cc @giorgiosironi

Reply:
Do they change across environments? Environment variables can, but should not, be changed at runtime (as in "during execution rather than in the application bootstrap").

Reply:
They're resolved during the bootstrap for a request, but not the application itself (when the container is compiled).

Reply:
So what's the alternative to environment variables, the configuration file? Seems to be similar in behavior.

Reply:
A config file, but it will only be read when compiling the container (ie needs a cache clear/redeploy to change).

Reply:
I'll keep it for now (to match the other), but probably something to review.